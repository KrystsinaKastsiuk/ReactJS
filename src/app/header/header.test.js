import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './header';
import Logo from './logo/logo';
import Search from './searchField/search';

describe('Header', () => {
  const wrapper = shallow(<Header />);

  it('has a logo', () => {
    expect(wrapper.find(Logo).exists()).toBe(true);
  });

  it('has a search', () => {
    expect(wrapper.find(Search).exists()).toBe(true);
  });
});


test('matches snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
