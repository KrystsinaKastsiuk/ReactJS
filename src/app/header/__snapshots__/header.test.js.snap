// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div />
  <div
    className="search"
  >
    <form
      className="searchField"
    >
      <h1
        className="searchTitle"
      >
        Find your movie
      </h1>
      <label>
        <input
          className="searchInput"
          placeholder="What do you need to find?"
          type="text"
        />
      </label>
      <button
        className="searchButton"
        type="submit"
      />
    </form>
    <form
      className="searchButtons"
    >
      <p
        className="searchBy"
      >
        Search by
      </p>
      <button
        className="searchByTitle"
        type="submit"
      >
        Title
      </button>
      <button
        className="searchByGenre"
        type="submit"
      >
        Genre
      </button>
    </form>
  </div>
</div>
`;
